context/app/static/js/components/workspaces/LaunchWorkspaceDialog/LaunchWorkspaceDialog.tsx

```diff
@@ -76,29 +79,38 @@ function LaunchWorkspaceDialog() {
     },
     [submit, handleClose, toastError],
   );
+
+  const dialogContent = (
```

Comment:
Generally, I think I'd prefer this to be in the jsx or its own component, but you don't have to address this now.